sdk/monitor/azure-opentelemetry-exporter-azuremonitor/README.md

```diff
@@ -114,5 +128,11 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 [opentelemetry_io]: https://opentelemetry.io/ 
 [span_data]: https://opentelemetry.lightstep.com/spans
 [sample_readme]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/monitor
-
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2Fsdk%monitor%2Fmicrosoft-opentelemetry-exporter-azuremonitor%2FREADME.png)
+[opentelemtry_spec]: https://opentelemetry.io/
+[instrumentation_library]: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/overview.md#instrumentation-libraries
+[tracer_provider]: https://opentelemetry-python.readthedocs.io/en/stable/api/trace.html?highlight=TracerProvider#opentelemetry.trace.TracerProvider
+[span_processor]: https://opentelemetry-python.readthedocs.io/en/stable/_modules/opentelemetry/sdk/trace.html?highlight=SpanProcessor#
```

Comment:
Are there non-`en` links available for this?

Reply:
Fixed.